src/tests/ftest/nvme/enospace.py

```diff
@@ -353,26 +351,32 @@ def test_performance_storage_full(self):
         # Write First
         self.start_ior_load(storage='SCM', operation="Auto_Write", percent=1)
         # Read the baseline data set
-        self.start_ior_load(storage='SCM', operation='Auto_Read', percent=1)
+        self.start_ior_load(storage='SCM', operation='Auto_Read', percent=1, create_cont=False)
+        baseline_container = self.nvme_local_cont
         max_mib_baseline = float(self.ior_matrix[0][int(IorMetrics.MAX_MIB)])
-        baseline_cont_uuid = self.ior_cmd.dfs_cont.value
         self.log.info("IOR Baseline Read MiB %s", max_mib_baseline)
 
         # Run IOR to fill the pool.
         self.run_enospace_with_bg_job()
 
         # Read the same container which was written at the beginning.
-        self.container.uuid = baseline_cont_uuid
-        self.start_ior_load(storage='SCM', operation='Auto_Read', percent=1)
-        max_mib_latest = float(self.ior_matrix[0][int(IorMetrics.MAX_MIB)])
-        self.log.info("IOR Latest Read MiB %s", max_mib_latest)
+        self.nvme_local_cont = baseline_container
+        # Add retry 10 loops with 60 seconds delay
+        for _loop in range(1, 11):
```

Comment:
We probably don't need this repeat loop.

Reply:
Just for testing propose. I will remove after test completed.